src/components/controls/dates/DatePicker/DatePicker.stories.tsx

```diff
@@ -95,17 +94,20 @@ Mouse.args = {
 Mouse.play = async ({ canvasElement }) => {
   const canvas = within(canvasElement)
   const input = canvas.getByRole("textbox", { name: "Label" })
+  let date = DateTime.now()
   await expect(input).toHaveValue("")
   await userEvent.click(input)
   const firstDayInCurrentMonthButton = canvas.getByRole("button", { name: "1" })
   await userEvent.click(firstDayInCurrentMonthButton)
-  await expect(input).toHaveValue(`${new Date().getFullYear()}-${new Date().getMonth() + 1}-01`)
+  date = date.set({ day: 1 })
+  await expect(input).toHaveValue(date.toFormat(DATE_FORMAT))
   await userEvent.click(input)
   const previousMonthButton = canvas.getAllByRole("button")[0]
   await userEvent.click(previousMonthButton)
   const firstDayInLastMonthButton = canvas.getByRole("button", { name: "1" })
   await userEvent.click(firstDayInLastMonthButton)
-  await expect(input).toHaveValue(`${new Date().getFullYear()}-${new Date().getMonth()}-01`)
+  date = date.minus({ month: 1 })
+  await expect(input).toHaveValue(date.toFormat(DATE_FORMAT))
```

Comment:
And like this, and similar in the other places?

```suggestion
  const firstDayInPreviousMonth = firstDayInCurrentMonth.minus({ month: 1 })
  await expect(input).toHaveValue(firstDayInPreviousMonth.toFormat(DATE_FORMAT))
```